### DISCUSSION TITLE
Add custom extensions for asset url imports?

### DISCUSSION THREAD

**Discussion by blaix** [remix-run/remix#1726] [Q&A]
I'm building a python wheel and trying to serve it like a public asset to be used with [pyodide](https://pyodide.org/en/stable/). Ideally I could do this by [importing it like a normal asset](https://remix.run/docs/en/v1/api/conventions#asset-url-imports):

```typescript
import wheel from "~/python/dist/my-wheel.whl";
```

But this errors with:

```
error: No loader is configured for ".whl" files: app/python/dist/my-wheel.whl
```

Is there a way to add ".whl" extension to use the esbuild `file` loader so they get added to `public/build` like normal assets?

**Comment by ccssmnn** [comment]
You could add it to the public folder and serve it from there. Why would you need to import it like this?

**Reply by blaix** [reply]
Yep, that's what I'm doing now. But looking for a way to handle it like I do some other imported assets. Not a huge deal but would be nice to have less special-case build code and have it fingerprinted for caching.
